proto/intellij_ide_info.proto

```diff
@@ -172,7 +172,12 @@ message Dependency {
   DependencyType dependency_type = 2;
 }
 
-// Next Available: 30
+// Contains the location of a Json encoded file containing Kotlin Toolchain Info..
+message KtToolchainIdeInfo {
+  ArtifactLocation location = 1;
```

Comment:
If there a more descriptive name we can use for this? (location is implied by the type, so maybe 'json_info_file' or similar would give more information?)

Reply:
In reply to both naming comments, Yes thats fine.

When I look at the other aspects `jvm`, `js` etc I plan to just piggy back over the same `json_info_file` -- one concern might be that native might not be well suited to IJWB. So I wonder if there is value in discriminating this in the model.